packages/react-storage/src/components/StorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import { createStorageBrowser } from '../createStorageBrowser';
+import * as controlsModule from '../context/controls';
```

Comment:
Prefer ordering the file to be asserted against  (.e.g. `../createStorageBrowser.tsx`) as the final import and  `PascalCase` for Module names

```suggestion

import * as ControlsModule from '../context/controls';
import { createStorageBrowser } from '../createStorageBrowser';
```